client/verta/tests/unit_tests/test_runtime.py

```diff
@@ -202,4 +202,14 @@ def test_s3_validation_good_keys(key: str):
     runtime._validate_s3(key)
 
 
-
+def test_overwrite_existing_key_throws_error() -> None:
+    """
+    If an attempt is made to write to the same key twice within a single
+    context, a KeyError exception is raised.
+    """
+    with runtime.context() as ctx:
+        with pytest.raises(ValueError)as err:
+                runtime.log('key_123', 'value_1')
+                runtime.log('key_123', 'value_2')
```

Comment:
As I mentioned in [my previous comment](https://github.com/VertaAI/modeldb/pull/3550#discussion_r1092569335), this should be

> _as specific as possible_

This isn't the case in your change here, because line `212` isn't expected to throw an error. Could we just use the suggestion from my comment?

Also there's extra unnecessary indentation here.

Reply:
Done.